packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -107,7 +108,10 @@ export const makeVaultManager = (
   // definition of reschedulePriceCheck, which refers to sortedVaultKits
   // XXX misleading mutability and confusing flow control; could be refactored with a listener
   /** @type {ReturnType<typeof makePrioritizedVaults>=} */
-  let prioritizedVaults;
+  let illiquidVaults;
+  /** @type {MapStore<string, InnerVault>} */
+  const vaultsToLiquidate = makeScalarBigMapStore('vaultsToLiquidate');
```

Comment:
I think it's worth mentioning that this is here to support  https://github.com/Agoric/agoric-sdk/issues/4568 .  Otherwise, the separate store doesn't seem to serve any purpose.